api/dp-openapi.yaml

```diff
@@ -784,6 +1132,163 @@ components:
         state:
           type: string
 
+    wordcheck_data:
+      type: object
+      properties:
+        langsWithDict:
+          type: array
+          items:
+            type: string
+        messages:
+          type: object
+        puncChars:
+          type: string
+
+    proof_page:
+      type: object
+      properties:
+        pagename:
+          type: string
+        image_url:
+          type: string
+        text:
+          type: string
+
+    page_text:
+      type: object
+      properties:
+        text:
+          type: string
+
+    round_data:
+      type: object
+      properties:
+        id:
+          type: string
+          example: F1
+        name:
+          type: string
+          description: the translated description
+          example: Formatting Round 1
+        formatting:
+          type: boolean
+          description: true if it is a formatting round
+          example: true
+        document:
+          type: string
+          description: url of the guidelines for the round
+          example: https://www.pgdp.net/wiki/DP_Official_Documentation:Formatting/Formatting_Guidelines
+
+    page_check:
+      description: check text using given languages and ad hoc good words
+      type: object
+      properties:
+        text:
+          type: string
+        auxlanguage:
+          type: string
+        adhocgoodwords:
+          type: array
+          items:
+            type: string
+
+    page_save:
+      type: object
+      properties:
+        new_page_state:
```

Comment:
we aren't consistent in camel case vs snake case in the properties on these objects

Reply:
We should use snake case everywhere in here to be consistent with the existing API.

Reply:
> We should use snake case everywhere in here to be consistent with the existing API.

I'll do that